UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -206,10 +206,10 @@ <h3>Metadata techniques</h3>
 			<h2>General information (supplement the bibliographic information)</h2>
 			<p>To solve the problem of displaying the accessibility metadata in a human readable form, vendors will determine their correct statement to display (from the User Experience Guide) by parsing the metadata and using the appropriate Display Techniques document. </p>
 
-			<p>The product details provide precious information about the usability of the book in relation to specific user needs. The following informations should always be displayed:</p>
+			<p>The product details provide precious information about the usability of the book in relation to specific user needs. The following information should always be displayed:</p>
 
 			<ul>
-				<li> File format (EPUB, PDF, MP3, Audiobook, etc.) </li>
+				<li> File format (EPUB, PDF, MP3, Audio book, etc.) </li>
```

Comment:
I'd undo this change. Audiobook is a pretty well established term. No need to add a space to it. Word has some strange ideas sometimes.
```suggestion
				<li> File format (EPUB, PDF, MP3, Audiobook, etc.) </li>
```